harness/determined/pytorch/deepspeed/dsat/_dsat_search_method.py

```diff
@@ -454,13 +458,14 @@ def on_validation_completed(
 
         # All DS AT Trials should be closed after validation.
         new_ops_list = [searcher.Close(request_id)]
```

Comment:
Is this not appending two Close operations? We also do it at the end of this function

Reply:
Yep, meant to delete the second one. Did it now.